prusa_slicer_post_processing_script.py

```diff
@@ -120,22 +121,21 @@ def main(gCodeFileStream,path2GCode,skipInput)->None:
             layer.addZ()
             layer.addHeight()
             lastfansetting=layer.spotFanSetting(lastfansetting)
-            layerobjs.append(layer)   
+            layerobjs.append(layer)
         for idl,layer in enumerate(layerobjs):
-            modify=False 
+            modify=False
             if idl<1:
                 continue # no overhangs in the first layer and dont mess with the setup
             else:
                 layer.extract_features()
                 layer.spotBridgeInfill()
                 layer.makePolysFromBridgeInfill(extend=parameters.get("ExtendIntoPerimeter",1))
                 layer.polys=layer.mergePolys()
-                layer.verifyinfillpolys()    
+                layer.verifyinfillpolys()
 
                 #ARC GENERATION
                 if layer.validpolys:
-                    modify=True
-                    gcodeWasModified=True
+                    numOverhangs += 1
```

Comment:
We track the number of overhangs for error printing if arc generation fails. Not required, but could be useful.